### DISCUSSION TITLE
6주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#19] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by narangd0** [comment]
제네릭에 대해 차근차근 알아가는 시간이 되어서 유익했다.
근데 다음주가 걱정된다.

퀴즈 타임 재밌었다.

**Reply by alanhakhyeonsong** [reply]
@narangd0 ㅊㅋ

**Comment by SuJeong68** [comment]
적당한 양이라 좋았고, 퀴즈 이벤트가 공부한 내용을 확인하기도 좋고 재밌었습니다🤣

**Comment by JeongJaew0n** [comment]
다들 실력이 느는게 보이네요. 남은 기간도 화이팅입니다!!

**Comment by alanhakhyeonsong** [comment]
예전엔 제네릭 이해도가 그닥 좋지 않았는데, 이번 기회에 제대로 알고가야겠다는 생각으로 다시 공부했더니 아주 쉽게 마스터했습니다.
(실력이 는건가...?)

**Comment by Kurmarkpzgr** [comment]
아이템 28에서 두 내용이 32, 33을 참조하라고 나왔는데, 다음주라 많이 기대됩니다.

**Comment by sdh46** [comment]
문장을 읽으면서 잘 못 이해한 부분이 있어 미궁 속으로 빠졌는데 다음에 이런 일이 있으면 다시 한번 차분히 읽고 생각해봐야겠다

**Comment by opgumo3** [comment]
전에 제네릭 공부를 했던 것이 도움이 됐던 것 같습니다.
이벤트성 질문으로 공부 내용 확인할 수 있어서 좋았습니다.
